api/tests/unit/domain/errors_test.js

```diff
@@ -63,6 +63,32 @@ describe('Unit | Domain | Errors', () => {
     expect(errors.UserNotAuthorizedToUpdateCampaignError).to.exist;
   });
 
+  describe('#SameNationalStudentIdInOrganizationError', () => {
+
+    it('should return a message with given nationalStudentId', () => {
```

Comment:
On pourrait utiliser des contexts pour donner un peu plus d'informations sur les deux cas de tests par exemple: 
```suggestion
    context('when the message with the national student id is given', () => { ... })
    context(''when no message is given, () => { ... })
```